Sources/Networking/Caching/CustomerInfoCallback.swift

```diff
@@ -28,3 +28,41 @@ struct CustomerInfoCallback: CacheKeyProviding {
     }
 
 }
+
+extension CustomerInfoCallback {
+
+    func withNewCacheKey(_ newKey: String) -> Self {
```

Comment:
Should this (or the extension) be `private`?

Reply:
Good call, dine,